mvn install fails on osx

Hello,

I am trying to build on latest osx and mvn fails as follows:

[INFO] --- frontend-maven-plugin:0.0.22:install-node-and-npm (install node and npm profile trigged) @ gateleen-hook-js ---
[INFO] Installing node version v0.12.2
[INFO] Creating temporary directory /Users/krital/dev/yachtsense/gateleen/gateleen-1.1.44/gateleen-hook-js/node_tmp
[INFO] Downloading Node.js from http://artifactory.pnet.ch/artifactory/list/ext-binaries-local/node/v0.12.2/node-v0.12.2-darwin-x64.tar.gz to /Users/krital/dev/yachtsense/gateleen/gateleen-1.1.44/gateleen-hook-js/node_tmp/node.tar.gz
[INFO] No proxy was configured, downloading directly
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for gateleen 1.1.44:
[INFO]
[INFO] gateleen ........................................... SUCCESS [  0.741 s]
[INFO] gateleen-testhelper ................................ SUCCESS [  1.778 s]
[INFO] gateleen-core ...................................... SUCCESS [ 14.681 s]
[INFO] gateleen-validation ................................ SUCCESS [  3.778 s]
[INFO] gateleen-logging ................................... SUCCESS [  3.910 s]
[INFO] gateleen-monitoring ................................ SUCCESS [  3.887 s]
[INFO] gateleen-routing ................................... SUCCESS [  5.184 s]
[INFO] gateleen-packing ................................... SUCCESS [  0.144 s]
[INFO] gateleen-delta ..................................... SUCCESS [  1.469 s]
[INFO] gateleen-queue ..................................... SUCCESS [ 10.274 s]
[INFO] gateleen-delegate .................................. SUCCESS [  3.601 s]
[INFO] gateleen-expansion ................................. SUCCESS [  3.208 s]
[INFO] gateleen-merge ..................................... SUCCESS [  0.213 s]
[INFO] gateleen-user ...................................... SUCCESS [  3.102 s]
[INFO] gateleen-security .................................. SUCCESS [  4.357 s]
[INFO] gateleen-scheduler ................................. SUCCESS [  3.706 s]
[INFO] gateleen-hook ...................................... SUCCESS [ 14.066 s]
[INFO] gateleen-hook-js ................................... FAILURE [  0.615 s]
[INFO] gateleen-qos ....................................... SKIPPED
[INFO] gateleen-player .................................... SKIPPED
[INFO] gateleen-runconfig ................................. SKIPPED
[INFO] gateleen-playground ................................ SKIPPED
[INFO] gateleen-test ...................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:19 min
[INFO] Finished at: 2019-04-12T20:27:00+03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:0.0.22:install-node-and-npm (install node and npm profile trigged) on project gateleen-hook-js: Could not download Node.js: Could not download http://artifactory.pnet.ch/artifactory/list/ext-binaries-local/node/v0.12.2/node-v0.12.2-darwin-x64.tar.gz: Unknown host artifactory.pnet.ch: nodename nor servname provided, or not known -> [Help 1]


Obviously the URL is invalid but as this is a maven (?) dependency I am unsure how to fix. Does it build for you somehow?

--- a/README.md
+++ b/README.md
@@ -26,6 +26,12 @@ You need Java 8 and Maven
 cd gateleen
 mvn install
 ```
+
+We're using a (private) mirror where we download npm/node from. In case you've
+no access to that mirror, you can use maven profile `publicRepos` (Eg: `mvn
+install -PpublicRepos`).
+
+
 ### Play
 
 A Redis is needed for this

